include/inexor/vulkan-renderer/vk_formats.hpp

```diff
@@ -0,0 +1,504 @@
+    case VK_FORMAT_B16G16R16G16_422_UNORM:
+        return "VK_FORMAT_B16G16R16G16_422_UNORM";
+    case VK_FORMAT_G16_B16_R16_3PLANE_420_UNORM:
+        return "VK_FORMAT_G16_B16_R16_3PLANE_420_UNORM";
+    case VK_FORMAT_G16_B16R16_2PLANE_420_UNORM:
+        return "VK_FORMAT_G16_B16R16_2PLANE_420_UNORM";
+    case VK_FORMAT_G16_B16_R16_3PLANE_422_UNORM:
+        return "VK_FORMAT_G16_B16_R16_3PLANE_422_UNORM";
+    case VK_FORMAT_G16_B16R16_2PLANE_422_UNORM:
+        return "VK_FORMAT_G16_B16R16_2PLANE_422_UNORM";
+    case VK_FORMAT_G16_B16_R16_3PLANE_444_UNORM:
+        return "VK_FORMAT_G16_B16_R16_3PLANE_444_UNORM";
+    case VK_FORMAT_PVRTC1_2BPP_UNORM_BLOCK_IMG:
+        return "VK_FORMAT_PVRTC1_2BPP_UNORM_BLOCK_IMG";
+    case VK_FORMAT_PVRTC1_4BPP_UNORM_BLOCK_IMG:
+        return "VK_FORMAT_PVRTC1_4BPP_UNORM_BLOCK_IMG";
+    case VK_FORMAT_PVRTC2_2BPP_UNORM_BLOCK_IMG:
+        return "VK_FORMAT_PVRTC2_2BPP_UNORM_BLOCK_IMG";
+    case VK_FORMAT_PVRTC2_4BPP_UNORM_BLOCK_IMG:
+        return "VK_FORMAT_PVRTC2_4BPP_UNORM_BLOCK_IMG";
+    case VK_FORMAT_PVRTC1_2BPP_SRGB_BLOCK_IMG:
+        return "VK_FORMAT_PVRTC1_2BPP_SRGB_BLOCK_IMG";
+    case VK_FORMAT_PVRTC1_4BPP_SRGB_BLOCK_IMG:
+        return "VK_FORMAT_PVRTC1_4BPP_SRGB_BLOCK_IMG";
+    case VK_FORMAT_PVRTC2_2BPP_SRGB_BLOCK_IMG:
+        return "VK_FORMAT_PVRTC2_2BPP_SRGB_BLOCK_IMG";
+    case VK_FORMAT_PVRTC2_4BPP_SRGB_BLOCK_IMG:
+        return "VK_FORMAT_PVRTC2_4BPP_SRGB_BLOCK_IMG";
+    case VK_FORMAT_ASTC_4x4_SFLOAT_BLOCK_EXT:
+        return "VK_FORMAT_ASTC_4x4_SFLOAT_BLOCK_EXT";
+    case VK_FORMAT_ASTC_5x4_SFLOAT_BLOCK_EXT:
+        return "VK_FORMAT_ASTC_5x4_SFLOAT_BLOCK_EXT";
+    case VK_FORMAT_ASTC_5x5_SFLOAT_BLOCK_EXT:
+        return "VK_FORMAT_ASTC_5x5_SFLOAT_BLOCK_EXT";
+    case VK_FORMAT_ASTC_6x5_SFLOAT_BLOCK_EXT:
+        return "VK_FORMAT_ASTC_6x5_SFLOAT_BLOCK_EXT";
+    case VK_FORMAT_ASTC_6x6_SFLOAT_BLOCK_EXT:
+        return "VK_FORMAT_ASTC_6x6_SFLOAT_BLOCK_EXT";
+    case VK_FORMAT_ASTC_8x5_SFLOAT_BLOCK_EXT:
+        return "VK_FORMAT_ASTC_8x5_SFLOAT_BLOCK_EXT";
+    case VK_FORMAT_ASTC_8x6_SFLOAT_BLOCK_EXT:
+        return "VK_FORMAT_ASTC_8x6_SFLOAT_BLOCK_EXT";
+    case VK_FORMAT_ASTC_8x8_SFLOAT_BLOCK_EXT:
+        return "VK_FORMAT_ASTC_8x8_SFLOAT_BLOCK_EXT";
+    case VK_FORMAT_ASTC_10x5_SFLOAT_BLOCK_EXT:
+        return "VK_FORMAT_ASTC_10x5_SFLOAT_BLOCK_EXT";
+    case VK_FORMAT_ASTC_10x6_SFLOAT_BLOCK_EXT:
+        return "VK_FORMAT_ASTC_10x6_SFLOAT_BLOCK_EXT";
+    case VK_FORMAT_ASTC_10x8_SFLOAT_BLOCK_EXT:
+        return "VK_FORMAT_ASTC_10x8_SFLOAT_BLOCK_EXT";
+    case VK_FORMAT_ASTC_10x10_SFLOAT_BLOCK_EXT:
+        return "VK_FORMAT_ASTC_10x10_SFLOAT_BLOCK_EXT";
+    case VK_FORMAT_ASTC_12x10_SFLOAT_BLOCK_EXT:
+        return "VK_FORMAT_ASTC_12x10_SFLOAT_BLOCK_EXT";
+    case VK_FORMAT_ASTC_12x12_SFLOAT_BLOCK_EXT:
+        return "VK_FORMAT_ASTC_12x12_SFLOAT_BLOCK_EXT";
+    case VK_FORMAT_MAX_ENUM:
+        return "0x7FFFFFFF";
```

Comment:
Is this correct?

Reply:
Nope. Another unnecessary value. I will remove that.

Reply:
Done.